ui/components/ui/metafox-logo/horizontal-logo.js

```diff
@@ -16,12 +16,14 @@ export default function MetaFoxHorizontalLogo({
       : themeProps,
   );
 
-  const fill = theme === 'dark' ? 'rgb(255,255,255)' : 'rgb(22,22,22)';
+  const fill = theme === ThemeType.dark ? 'rgb(255,255,255)' : 'rgb(22,22,22)';
 
   useEffect(() => {
-    if (themeProps !== undefined) {
-      setTheme(themeProps);
+    let newTheme = themeProps;
+    if (newTheme === undefined) {
+      newTheme = document.documentElement.getAttribute('data-theme');
     }
+    setTheme(newTheme);
```

Comment:
I think we could refactor this component to use CSS variables rather than having to fetch theme state but it can be done in a subsequent PR